packages/core/src/modules/connections/services/TrustPingService.ts

```diff
@@ -20,6 +39,16 @@ export class TrustPingService {
 
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   public processPingResponse(inboundMessage: InboundMessageContext<TrustPingResponseMessage>) {
-    // TODO: handle ping response message
+
+    const { agentContext, connection } = inboundMessage
```

Comment:
You can do:

```
const connection = inboundMessage.assertReadyConnection()
```

The trust ping handler already checks whether there is a connection, so this should never error but if the handler ever changes makes sure an error is thrown instead of it being silently ignored

Reply:
I'm not actually 100% sure that we need to require the connection to be complete to send the trust ping, so perhaps this actually isn't needed.  🤔

Reply:
In what cases is a connection not required? We can only send trust ping messages to completed connections, so if we were to receive a response we MUST have a connection, otherwise the ping response is not in response to a ping we sent

Reply:
Fixed